store/sqlstore/user_store.go

```diff
@@ -325,6 +325,25 @@ func (us SqlUserStore) UpdateMfaActive(userId string, active bool) error {
 	return nil
 }
 
+// GetMany returns a list of users for the provided list of ids
+func (us SqlUserStore) GetMany(ids []string) ([]*model.User, error) {
+	query := us.usersQuery.Where(sq.Eq{"Id": ids})
+	queryString, args, err := query.ToSql()
+	if err != nil {
+		return nil, errors.Wrap(err, "users_get_many_tosql")
+	}
+
+	var users []*model.User
+	if _, err := us.GetReplica().Select(&users, queryString, args...); err != nil {
+		if errors.As(err, sql.ErrNoRows) {
```

Comment:
This should be `errors.Is` I believe.

Also, any reason to check for no rows in a range query? We are not doing this anywhere else in user_store.

Reply:
Totally 👍 . I've handled the error, because I think is more reasonable for this methdo to return an empty slice when the query didn't return anything(in a sense treating that error as a non-error)

Reply:
Yes, `Select` does a `Query`, so it will return zero rows if it doesn't find one.